### ISSUE TITLE
urllib.error.URLError raised when executing

### ISSUE THREAD

**Issue by Morpheus636** [Morpheus636/redownload#31]
```
Enter the Relisten or Archive.org URL to download the tracks from: https://relisten.net/grateful-dead/1977/05/09/help-on-the-way?source=337173
Enter the path to save the tracks to (ENTER for default [./redownlaods]): 
Traceback (most recent call last):
  File "urllib/request.py", line 1346, in do_open
  File "http/client.py", line 1285, in request
  File "http/client.py", line 1331, in _send_request
  File "http/client.py", line 1280, in endheaders
  File "http/client.py", line 1040, in _send_output
  File "http/client.py", line 980, in send
  File "http/client.py", line 1454, in connect
  File "ssl.py", line 500, in wrap_socket
  File "ssl.py", line 1040, in _create
  File "ssl.py", line 1309, in do_handshake
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "redownload_cli.py", line 41, in <module>
  File "redownload_cli.py", line 37, in main
  File "redownload/__init__.py", line 11, in redownload
  File "redownload/web_parsing.py", line 16, in html_from_url
  File "urllib/request.py", line 214, in urlopen
  File "urllib/request.py", line 517, in open
  File "urllib/request.py", line 534, in _open
  File "urllib/request.py", line 494, in _call_chain
  File "urllib/request.py", line 1389, in https_open
  File "urllib/request.py", line 1349, in do_open
urllib.error.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)>
[64272] Failed to execute script 'redownload_cli' due to unhandled exception! 
```

**Comment by Morpheus636** [comment]
Looks like an SSL issue with mac, since I can't reproduce on linux. According to [this](https://medium.com/@yen.hoang.1904/resolve-issue-ssl-certificate-verify-failed-when-trying-to-open-an-url-with-python-on-macos-46d868b44e10), it can be resolved by installing certifi.

I'm also going to make it so tests run on all supported OS as part of this fix.

**Comment by Morpheus636** [comment]
Update: Still an issue in v0.1.1a1, reopening.

**Comment by Morpheus636** [comment]
Still present as of v0.1.1a2

**Comment by Morpheus636** [comment]
Seems to be fixed as of v0.1.1a4
